cloud/linode/linode.py

```diff
@@ -446,6 +463,8 @@ def main():
             api_key = dict(),
             name = dict(type='str'),
             plan = dict(type='int'),
+            stack_script_id = dict(type='int'),
+            stack_script_udf_responses = dict(type='str'),
```

Comment:
Ah, yes, my test environment had not been updated to latest. No longer getting the unknown type error. Now getting: `"unsupported parameter for module: stack_script_udf_responses"`. I am not sure how to interpret that message.